### ISSUE TITLE
Bug: Sub-nav menu is not accessible on Safari

### ISSUE THREAD

**Issue by idealhack** [kubernetes/website#8843]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**

This is a screenshot on Safari 11.1 / macOS 10.13.4, It works fine on latest Google Chrome. I've seen this maybe for a year. I tried to fix this several times but didn't make it.

![image](https://user-images.githubusercontent.com/408385/40787290-d1fcf2de-651f-11e8-9688-82aa27869d84.png)

**Proposed Solution:**

**Page to Update:**

Basically all pages of https://kubernetes.io

**Comment by Sai-Adarsh** [comment]
Still a bug? shall I work on this? @idealhack

**Comment by idealhack** [comment]
@Sai-Adarsh I believe so, feel free to take it :)

**Comment by sftim** [comment]
/priority backlog
Is that OK?

**Comment by idealhack** [comment]
I thought bugs shouldn't be backlog? this is a long-existing bug though.

thanks for triaging issues in this repo.

**Comment by hvaara** [comment]
Still an issue
![image](https://user-images.githubusercontent.com/1535968/66084153-38959080-e56e-11e9-8313-29629d3277fa.png)

**Comment by hvaara** [comment]
Thanks for looking at this @furukawa3! 🎉 In order for the bot to recognize commands each one has to be on a new line. I took the liberty of assigning you

/assign @furukawa3 
/lifecycle active

**Comment by furukawa3** [comment]
Oops!!  Thanks @hvaara for assigning :)

**Comment by furukawa3** [comment]
1 question.  Could you tell me what kind of browser does k8s document page support?  Currently, I try to fix Safari first.  ( I also found the similar error with IE 11.0.146.  Only "language" Sub-nav was wrong. )

![ie11](https://user-images.githubusercontent.com/8042145/66887498-2b6cad00-f016-11e9-8edd-eccc57fa342d.png)

**Comment by idealhack** [comment]
@furukawa3 san, thanks for looking at this!

From what I can confirm, this sub-sav works with Chrome and FireFox, but not work with Safari.

**Comment by idealhack** [comment]
it seems this was fixed after the theme update: https://kubernetes.io/blog/2020/06/better-docs-ux-with-docsy/
